### ISSUE TITLE
Further exploration of benzylic alcohols (and a faster route to them)

### ISSUE THREAD

**Issue by david1597** [OpenSourceMalaria/Series4#44]
The potent Pfizer compound hints at the low potency from introducing a benzylic hydoxyl group on the LHS. Taking a look at the full series four library there are 11 compounds which have an oxygen atom in this benzylic position and all have a potency of <800 nM. I am currently adding more to this library in the form of the OHOH compounds, although a drawback to this synthesis is the number of steps (five) to make a unique compound: 
![image](https://user-images.githubusercontent.com/30584472/40211129-2d20dfe8-5a8c-11e8-8f87-bfbbe55c66a8.png)

It'd be good to have a faster route to look at the effects of varying ring substitution in this position, or even explore different rings. This should be possible in the one-step process outlined in the following scheme. It starts with the -OH substituted core (which I have a smallish amount of as a side-product of previous reactions) and reacting with chloroacetone to get a ketone. It is then just a single step from that intermediate to generate a library of final compounds, using any Grignard reagent that can be prepared.
![image](https://user-images.githubusercontent.com/30584472/40210546-d47ad41e-5a88-11e8-89cc-4e9a3fc90e63.png)

**Comment by MFernflower** [comment]
Fantastic idea! It might be interesting to make one or two deplanarized compounds from say maybe cyclohexyl grignard and cyclobutyl grignard? (or whatever cycloalkyl grignard you have to hand)

**Comment by david1597** [comment]
Yep, it opens up lots of possibilities for quickly assessing effects of that most northwestern corner. 

To test this route I'm initially using the material recovered as a side product which should be about 100-150 mg of the ketone. That will be enough to make 2 or 3 final compounds. To chose these 2/3, I've looked back through the library at the hits to date. It's overwhelmingly the 3,4-difluoro substitution. There are the two compounds shown below however, so I'm proposing to start with the  _o_-Cl and _m_-OMe Grignards. This way we may reasonably expect the new compounds to show some potency rather than starting with something wildly different. If it works, well, then we can get adventurous and rapidly expand the library.

![image](https://user-images.githubusercontent.com/30584472/40222812-2047cec0-5ac4-11e8-9f32-60b4dc309709.png)

**Comment by MFernflower** [comment]
I'm curious why choose to use features from compounds with greater than 100
nanomolar ic50's

**Comment by mattodd** [comment]
@MFernflower just because the expectation is that potency is cumulative. i.e. those motifs might help a little.

**Comment by MedChemProf** [comment]
While @david1597 is working out the conditions for what looks to be an efficient route to new benzylic alcohol analogs, we were interested in synthesizing the amino variant through the traditional route. To be consistent, we will also make the tertiary alcohol as a comparison:
![new_scheme_temp1](https://user-images.githubusercontent.com/14179120/40309279-ecf8dd60-5cd6-11e8-89ca-0af190a1d5c2.jpg)

![new_scheme_3](https://user-images.githubusercontent.com/14179120/40309313-0818a422-5cd7-11e8-9317-867b744ba8a3.jpg)

We are also planning on gearing up to revisit some of our scaffold hop ideas using the below sequence:
![new_scheme_temp](https://user-images.githubusercontent.com/14179120/40309372-304504ae-5cd7-11e8-831b-e39c5f1c4d64.jpg)

The semester is just starting here and I do not have any research students at the moment. I am working on remedying that situation.

**Comment by MFernflower** [comment]
Interesting proposal @MedChemProf but may I nitpick at something minor? Why use the more expensive TBDMS di-protected dihydroxyacetone when you can protect the DHA with cheaper TES?

**Comment by MedChemProf** [comment]
The differential cost is inconsequential over the entire synthesis and it is a more robust protecting group.

**Comment by david1597** [comment]
Great plan @MedChemProf. I had [issues ](#41 )coupling the triol to the core and needed to protect to get this to work. I now have the RHS difluoromethoxy version of the compound you propose above. Awaiting high res characterisation data and I'll update [here ](#36)when that is confirmed. 

@MFernflower As Mat said for the choice of compounds above. 9 out of the 10 compounds with <100 nM potency have the 3,4-difluoro substitution (the other compound is Ed's carborane). My two suggestions above are the next best thing from the database with the same core structure, and so the best starting point to see whether introduction of the benzylic alcohol leads to an improvement.

**Comment by MFernflower** [comment]
I assume di fluoro is top priority then?

>

**Comment by david1597** [comment]
@MFernflower well it's certainly present in our most potent molecules to-date, but is this partly because 1 in 3 compounds from our library contain a 3,4-difluorobenzene? It's not an area that has been well explored so far.

The aim of this thread was to explore this area and see if something else is better. And that particular structure was chosen partly because the synthesis is one step, but also because the difluoro substitution is already in the library - with 101 nM potency - so we have our benchmark for comparison.

**Comment by david1597** [comment]
This synthetic route is on temporary hold.

The starting material was not the compound indicated above. The -OH is actually at the 8-position, as we've just [discovered ](#46)in a number of cases. The reaction with chloroacetone was successful to give the corresponding 8-propan-2-one. Details at [DGS 78-1](https://au-mynotebook.labarchives.com/share/David%2520Smith%2520OSM/MzMxLjV8MzAxNzEvMjU1L1RyZWVOb2RlLzMwNDg2NzUxMDh8ODQxLjU=). 
![image](https://user-images.githubusercontent.com/30584472/40464547-322d4c2e-5f5f-11e8-9a63-5a1503a82207.png)


_Strings
DGS 78-1
OSM-S-561
FC(F)OC(C=C1)=CC=C1C2=NN=C3C(OCC(C)=O)=NC=CN32
InChI=1S/C15H12F2N4O3/c1-9(22)8-23-14-13-20-19-12(21(13)7-6-18-14)10-2-4-11(5-3-10)24-15(16)17/h2-7,15H,8H2,1H3
MXHWALUHBSFDFZ-UHFFFAOYSA-N_

**Comment by david1597** [comment]
@MedChemProf I've moved your benzylic amine idea to #50 and will add some comments there
